src/routes/getFiles.ts

```diff
@@ -0,0 +1,235 @@
+import {
+  asBoolean,
+  asMap,
+  asNumber,
+  asObject,
+  asOptional,
+  asString
+} from 'cleaners'
+import Router from 'express-promise-router'
+
+import config from '../../config.json'
```

Comment:
Use fs over imports on config.json. This allows for a runtime error to be thrown for missing config.json instead of a compile error after cloning the project.